docs/installation/generic.md

```diff
@@ -2,12 +2,13 @@
 
 ### Requirements
 
-- Operating system Ubuntu 14.04 or above is preferred. (This guide will be based on ubuntu/debian based operating systems.)
```

Comment:
Please don't remove the disclaimer about the guide being mainly for linux as this platform is what we support officially, and might not be able to test and support OSX for each change or release